.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+
+on: 
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
+      - name: Cache Maven and Gradle
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.m2/repository
+            ~/.gradle
+          key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
+      - name: Build
+        run: |
+          mvn clean install -U
```

Comment:
add "-B" option, also in call below

Reply:
what does this do? is no there in original one

Reply:
```
 -B,--batch-mode                        Run in non-interactive (batch)
                                        mode (disables output color)
```

Doesn't github action log support colors?

Reply:
it reduces the noise of download messages